src/components/MainNav/Notifications/NotificationsMenu.js

```diff
@@ -102,8 +102,8 @@ class NotificationsMenu extends React.Component {
     return (
       <Row key={notification.id} onClick={() => { this.onClickNotification(notification); }} className={this.getNotificationClass(notification.date)}>
         <Col xs>
-          <div><strong>{notification.source} {formattedDate}</strong></div>
-          <p>{text} ({link})</p>
+          {`<div><strong>${notification.source} ${formattedDate}</strong></div>`}
+          {`<p>${text} (${link})</p>`}
```

Comment:
Same comment here about string interpolation. I don't think JSX would properly be evaluated inside of template literals. And even the native DOM elements are actually special JSX components under the hood.